invoices/invoiceregistry_test.go

```diff
@@ -137,10 +155,10 @@ func TestSettleInvoice(t *testing.T) {
 	hodlChan := make(chan interface{}, 1)
 
 	// Try to settle invoice with an htlc that expires too soon.
-	event, err := registry.NotifyExitHopHtlc(
+	event, err := ctx.registry.NotifyExitHopHtlc(
 		hash, testInvoice.Terms.Value,
 		uint32(testCurrentHeight)+testInvoiceCltvDelta-1,
-		testCurrentHeight, getCircuitKey(10), hodlChan, nil,
+		testCurrentHeight, getCircuitKey(10), hodlChan, testPayload,
```

Comment:
why this change from nil to `testPayload`?

Reply:
Added commit message